src/js/media-text-display.js

```diff
@@ -56,7 +56,6 @@ template.innerHTML = /*html*/`
  *
  * @cssproperty --media-control-display - `display` property of control.
  * @cssproperty --media-control-background - `background` of control.
- * @cssproperty --media-control-hover-background - `background` of control hover state.
```

Comment:
oh since you deleted here, it's needed in media-time-display

Reply:
ah, good catch

Reply:
Updated